### ISSUE TITLE
Referencing a value inside a yaml file

### ISSUE THREAD

**Issue by vondacho** [WindomZ/swagger-merger#71]
Given the following input file: `in.yaml`
```
description:
    $ref: ./desc.yaml#/description
```
And the following referenced file: `desc.yaml`:
```
description: this is a value
```
When I merge the input file using `swagger-merger -i in.yaml -o out.yaml `

Then I receive the following error: `Unexpected token } in JSON at position 33`

**Comment by WindomZ** [comment]
Thanks for your feedback!
This bug has been fixed, now you can update to the latest version to verify.

**Comment by WindomZ** [comment]
Issue closed in version v1.5.0.
